docs/university/core-components/collections.md

```diff
@@ -0,0 +1,44 @@
+# 🗄️ Collections
```

Comment:
component is called Collection, singular, that means we need to change it everywhere: file name, title, content